numba/core/typing/context.py

```diff
@@ -307,7 +307,7 @@ def resolve_setattr(self, target, attr, value):
     def resolve_static_getitem(self, value, index):
         assert not isinstance(index, types.Type), index
         args = value, index
-        kws = ()
+        kws = {}
```

Comment:
Is there a measurable impact from altering this?

Reply:
Probably no, I thought it was the reason of some test fails, but seems not, so I've reverted it.